docs/data/material/discover-more/roadmap/roadmap.md

```diff
@@ -47,41 +47,37 @@ Here are the components we will work on being supported in the MUI ecosystem:
-| Advanced Layout                                                | MUI X    | ⏳     |
-| Carousel                                                       | MUI X    | ⏳     |
-| [Charts](https://mui.com/x/react-charts/)                      | MUI X    | 🧪     |
-| [Data Grid](/x/react-data-grid/)                               | MUI X    | ✅     |
-| [Date Picker](/x/react-date-pickers/date-picker/)              | MUI X    | ✅     |
-| [Time Picker](/x/react-date-pickers/time-picker/)              | MUI X    | ✅     |
-| [Date Time Picker](/x/react-date-pickers/date-time-picker/)    | MUI X    | ✅     |
-| [Date Range Picker](/x/react-date-pickers/date-range-picker/)  | MUI X    | ✅     |
-| Time Range Picker                                              | MUI X    | ⏳     |
-| Date Time Range Picker                                         | MUI X    | ⏳     |
-| Dropdown                                                       | MUI Core | ⏳     |
-| Dropzone                                                       | MUI X    | ⏳     |
-| File Upload                                                    | MUI X    | ⏳     |
-| Gantt Chart                                                    | MUI X    | ⏳     |
-| Gauge                                                          | MUI X    | ⏳     |
-| Image                                                          | MUI Core | ⏳     |
-| [Masonry](/material-ui/react-masonry/)                         | MUI Core | 🧪     |
-| Navbar                                                         | MUI Core | ⏳     |
-| Nested Menu                                                    | MUI X    | ⏳     |
-| NProgress                                                      | MUI Core | ⏳     |
-| Numeric Input                                                  | MUI Core | ⏳     |
-| Rich Text Editor                                               | MUI X    | ⏳     |
-| Scheduler                                                      | MUI X    | ⏳     |
-| Scrollspy                                                      | MUI Core | ⏳     |
-| Sparkline                                                      | MUI X    | ⏳     |
-| [Timeline](/material-ui/react-timeline/)                       | MUI Core | 🧪     |
-| Tree select                                                    | MUI X    | ⏳     |
-| [Tree View](/x/react-tree-view/)                               | MUI X    | 🧪     |
-| Tree View - Checkbox                                           | MUI X    | ⏳     |
-| Tree View - Drag & Drop                                        | MUI X    | ⏳     |
-| [Tree View - Multiselect](/x/react-tree-view/#multi-selection) | MUI X    | 🧪     |
-| Tree View - Virtualization                                     | MUI X    | ⏳     |
-| Window Splitter                                                | MUI X    | ⏳     |
+| Name                                                          | Product  | Status |
```

Comment:
Not sure why the entire table got diffed.
I just removed the nested tree view items and marked the tree view as stable.

Reply:
I think it's because `[Tree View - Multiselect](/x/react-tree-view/#multi-selection)` was removed, reducing the width of the first column.